### PR TITLE
Minor bug fix. Typo in ETX formatter

### PR DESCRIPTION
**Pull request by robschleusner**

### CODE REVIEW

File: packages/providers/lib.esm/base-provider.js [hunk 1]
```diff
@@ -800,9 +800,9 @@ export class BaseProvider extends Provider {
         });
     }
     poll() {
-        // Polling is now disabled
-        return;
         return __awaiter(this, void 0, void 0, function* () {
+            // Polling is now disabled
+            return;
```
Comment (wizeguyy) [inline packages/providers/lib.esm/base-provider.js:805]: any idea why this changed?

File: packages/quais/dist/quais.umd.js [hunk 2]
```diff
@@ -25573,7 +25573,7 @@
 	        _this._maxInternalBlockNumber = -1024;
 	        _this._lastBlockNumber = -2;
 	        _this._maxFilterBlockRange = 10;
-	        _this._pollingInterval = 4000;
+	        _this._pollingInterval = 40000;
```
Comment (wizeguyy) [inline packages/quais/dist/quais.umd.js:25576]: I don't think you meant to change this

Comment (alanorwick) [reply packages/quais/dist/quais.umd.js:25576]: same as above

File: packages/providers/lib.esm/base-provider.js (hunk 1 above)
Comment (alanorwick) [reply packages/providers/lib.esm/base-provider.js:805]: I think this is a left over compile from Jonathans commit

Comment (alanorwick) [reply packages/providers/lib.esm/base-provider.js:805]: this [commit](https://github.com/dominant-strategies/quais-5.js/commit/eab3e2280441f8d1648448ccdef4e24acfd78f07#diff-3729c4e3b80d703fe53e6558c7300d1a5a43ba468e047490ee72c37b50b4e53a) didn't touch lib.esm/base-provider.js

So I'm assuming a different build command ran?

**Review by alanorwick — APPROVED** [review]

File: packages/providers/lib.esm/base-provider.js (hunk 1 above)
Comment (wizeguyy) [reply packages/providers/lib.esm/base-provider.js:805]: yeah, Jonathan introduced the original code. I'm not sure why it would've changed in this PR

Comment (wizeguyy) [reply packages/providers/lib.esm/base-provider.js:805]: @jdowning100 can you confirm the above change does what you want it to?

The hypothesis atm is that you hadn't compiled the latest version of `base-provider.ts`, and now that @robschleusner is compiling, he sees changes to your code. Seems likely to me, but either way, I want you to confirm the compiled result is what you intended.

Comment (jdowning100) [reply packages/providers/lib.esm/base-provider.js:805]: I thought I had compiled it with `npm run build` but perhaps that doesn’t work? In any case, this looks to be correct.

Comment (jdowning100) [reply packages/providers/lib.esm/base-provider.js:805]: Perhaps in the future we should gitignore built/compiled js files?

Comment (wizeguyy) [reply packages/providers/lib.esm/base-provider.js:805]: > Perhaps in the future we should gitignore built/compiled js files?

I'm open to this

**Review by wizeguyy — APPROVED** [review]

**Pull request merged**